### DISCUSSION TITLE
Can I do fine-tuning Stable Diffusion 3 with LoRA?

### DISCUSSION THREAD

**Discussion by victolee0** [huggingface/diffusers#8891] [Q&A] [answered]
I want to do fine-tuning Stable Diffusion 3 with LoRA, without Dreambooth.
The examples of diffusers are included in Dreambooth.
How do I fine-tuning Stable Diffusion 3 with LoRA only?

**Comment by asomoza** [comment, marked as answer]
Hi, the training scripts are for learning and to adapt them to your specific task. We don't have the training scripts for everything, you can adapt the Dreambooth Lora one to not use Dreambooth and you can learn how to do it from the SDXL example.

Also for serious fine-tuning we recommend to use [SimpleTuner](https://github.com/bghira/SimpleTuner) which also uses Diffusers and has the integration you want plus a lot more.

**Reply by victolee0** [reply]
thank you!
